source/frontend/src/features/contacts/contact/edit/Organization/UpdateOrganizationForm.tsx

```diff
@@ -83,12 +83,16 @@ const UpdateOrganization: React.FC<FormikProps<IEditableOrganizationForm>> = ({
 
   const organizationId = getIn(values, 'id');
   const persons = getIn(values, 'persons') as Partial<ApiGen_Concepts_Person>[];
+  const isOrganizationDisabled = getIn(values, 'isDisabled');
 
   const onCancel = () => {
     history.push(`/contact/O${organizationId}`);
   };
 
   const isContactMethodInvalid = useMemo(() => {
+    if (isOrganizationDisabled === 'true') {
```

Comment:
Feels weird that this should be a boolean but the rigth hand operator is a string?

Reply:
This is because Formik values for dropdowns (SELECT input field) are always strings. We have several utility methods to convert from "stringy" booleans to "real" booleans

Reply:
let me try with with booleans. I see the tranform method when it is saved and send to the API but not before that.